docs/proposals/003-model-server-protocol/protocol.md

```diff
@@ -84,23 +68,5 @@ And the response MAY look like this:
 }
 ```
 
-#### Dynamically Register/Unregister Adapters
-
-Model servers SHOULD have APIs to dynamically register/unregister models (usually LoRA adapters).
-This enables platform teams to multiplex multiple LoRA adapters on shared model servers and
-dynamically rollout LoRA adapters. 
-
-NOTE this is not a strict requirement from the inference extension, but a critical feature for CI/CD
-integration.
-
-While we don’t intend to dictate how model servers should implement this API, a reference REST API
-MAY look this:
-
-```
-POST ${server_endpoint}/adapters/{adapter-id}
-{
-        "path": "path/to/my/adapter"
-}
-
-DELETE ${server_endpoint}/adapters/{adapter-id}
-```
+NOTE: Currently in vLLM v0.6.6, LoRA info is exposed in the `vllm:lora_requests_info` metric, where
+`MaxActiveAdapters` is exposed as a string label `max_lora`, and `ActiveAdapters` as a comma separated string label `running_lora_adapters`.
```

Comment:
Are these differences from the standard defined in L45-L51 being resolved in the vLLM community? If so, please provide a link to the appropriate GitHub issue.

Reply:
Added the tracking issue for vllm integration, and will update the tracking issue with the new requirement.